packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -2425,3 +2334,119 @@ QUnit.module('drag text', moduleConfig, () => {
         assert.strictEqual(event.isDefaultPrevented(), true, `the ${DROP_EVENT_NAME} event is prevented`);
     });
 });
+
+QUnit.module('format: "," as a decimal separator', moduleConfig, () => {
+    function executeActionWithSuppressErrors(action) {
+        const logErrorsStub = sinon.stub(errors, 'log');
+        action();
+        logErrorsStub.restore();
+    }
+
+    QUnit.test('value option should have right value after inserting when format is enabled and decimalSeparator is \',\' (T829935)', function(assert) {
+        const oldDecimalSeparator = config().decimalSeparator;
+
+        executeActionWithSuppressErrors(() => config({ decimalSeparator: ',' }));
```

Comment:
let's minify copypaste. I suggest to move  the change of the decimal separator to beforeEach/afterEach. It will also make possible to get rid of the `try/finally` construction.